e2e/cypress/tests/integration/account_settings/profile/fullname_truncate_spec.ts

```diff
@@ -21,7 +21,7 @@ describe('Profile > Profile Settings> Full Name', () => {
     before(() => {
         cy.apiInitSetup().then(({team, user, offTopicUrl}) => {
             firstUser = user;
-            cy.apiCreateUser().then(({user: user1}) => {
+            cy.apiCreateUser({}).then(({user: user1}) => {
```

Comment:
I would suggest to update the type at https://github.com/mattermost/mattermost-webapp/blob/f5aa82d04ad05d74d60ea359188f2c6afd715d57/e2e/cypress/tests/support/api/user.d.ts#L208 to indicate the exact optional parameter, like:
```diff
- apiCreateUser(options: Record<string, any>): Chainable<{user: UserProfile}>;
+ apiCreateUser(options?: {
+     user?: Partial<UserProfile>;
+     prefix?: string;
+     bypassTutorial?: boolean;
+     showOnboarding?: boolean;
+ }): Chainable<{user: UserProfile}>;
```

so that this line will remain as `cy.apiCreateUser()...`

Reply:
Looking at this as resolved but can't see related change, are you planning to address this here or separately? I'm fine either way, please let me know. TIA